project/src/components/header/header.tsx

```diff
@@ -11,9 +12,7 @@ function Header({ hasNav = true }: HeaderProps): JSX.Element {
       <div className="container">
         <div className="header__wrapper">
           <div className="header__left">
-            <Link className="header__logo-link header__logo-link--active" to={AppRoute.Main}>
-              <img className="header__logo" src="img/logo.svg" alt="6 cities logo" width="81" height="41" />
-            </Link>
+            <Logo type='header' />
```

Comment:
В jsx атрибуты обычно в двойных кавычках 
```suggestion
            <Logo type="header" />
```